src/store/store.ts

```diff
@@ -1451,45 +1456,55 @@ export const useStore = defineStore('store', {
     },
     setAmalgamationCourtApproval (courtApproval: boolean) {
       this.stateModel.amalgamation.courtApproval = courtApproval
+      if (!this.stateModel.ignoreChanges) this.stateModel.haveChanges = true
     },
     setAmalgamationCourtApprovalValid (valid: boolean) {
       this.stateModel.amalgamation.courtApprovalValid = valid
     },
     setAmalgamationType (type: AmalgamationTypes) {
       this.stateModel.amalgamation.type = type
+      if (!this.stateModel.ignoreChanges) this.stateModel.haveChanges = true
     },
-    setRestorationType (type: RestorationTypes) {
-      this.stateModel.restoration.type = type
-    },
-    setRestorationExpiry (expiry: string) {
-      this.stateModel.restoration.expiry = expiry
-    },
-    setRestorationRelationships (relationships: RelationshipTypes[]) {
-      this.stateModel.restoration.relationships = relationships
-    },
-    setBusinessNameValid (valid: boolean) {
-      this.stateModel.restoration.businessNameValid = valid
-    },
-    setRestorationTypeValid (valid: boolean) {
-      this.stateModel.restoration.restorationTypeValid = valid
+
+    //
+    // restoration-specific setters
+    //
+    setRestorationApplicationDate (date: string) {
+      this.stateModel.restoration.applicationDate = date
+      if (!this.stateModel.ignoreChanges) this.stateModel.haveChanges = true
     },
     setRestorationApprovalType (type: ApprovalTypes) {
       this.stateModel.restoration.approvalType = type
+      if (!this.stateModel.ignoreChanges) this.stateModel.haveChanges = true
+    },
+    setRestorationApprovalTypeValid (valid: boolean) {
+      this.stateModel.restoration.approvalTypeValid = valid
     },
-    setRestorationCourtOrderNumber (fileNumber: string) {
```

Comment:
This is obsolete? We don't need it anymore?

Reply:
Wasn't used anywhere.

There's another setter to save the court order object, which is used.

Reply:
Thanks, resolving.